dask/dataframe/groupby.py

```diff
@@ -87,7 +87,11 @@ def _maybe_slice(grouped, columns):
     """
     Slice columns if grouped is pd.DataFrameGroupBy
     """
-    if isinstance(grouped, pd.core.groupby.DataFrameGroupBy):
+    # generalized approach of groupby checking
+    # cudf and pandas currently do not have much overlap in
+    # attribute functionality
+    typ = type(grouped)
+    if bool('groupby' in typ.__name__.lower()):
```

Comment:
Style nits: 

-  You can probably drop the `bool`  
- You might also consider inlining the `type` call.